MANIFEST.in

```diff
@@ -8,3 +8,4 @@ recursive-include examples *.py
 prune docs/_build
 prune docs/gh-pages
 include numba/_version.py
+
```

Comment:
unnecessary changes.

Reply:
:) oh yes, thanks, that was from me trying to force in some data files that I changed my mind about, clearly left a space. Will fix.